requirements.txt

```diff
@@ -34,8 +34,13 @@ SQLAlchemy==2.0.23
 SQLAlchemy-Utils==0.41.1
 tables==3.9.1
 urllib3==2.1.0
+uv==0.4.23
 wget==3.2
 
+# NOTE : pytorch is also required, but I haven't figured out how to get
+# a CPU-only version of pytorch from PyPi.  (The GPU version adds 5GB to
+# the Dockerfile...!)   So, it's manually installed in the Dockerfile.
+
```

Comment:
This is 5GB of storage on the machine running the pipeline as opposed to 5GB ram usage per running instance of the pipeline, right?

Reply:
...you are pushing the limits of my docker knowledge.

It's 5GB of storage on the machine plus 5GB of RAM on the running machine, but I don't think it's 5GB for running container, because I believe multiple containers can share the memory used to store the image.

In any event, it's ridiculous, since the docker image is 3GB without the GPU stuff, and we aren't using the GPU at all.